packages/ui-components/src/components/UIComboBox/UIComboBox.tsx

```diff
@@ -165,6 +180,18 @@ export class UIComboBox extends React.Component<UIComboBoxProps, UIComboBoxState
         return true;
     }
 
+    /**
+     * This is a workaround for FluentUI not handling aria-invalid correctly for a ComboBox.
+     * For the time being we add logic here to set aria-invalid based on the presence of
+     * an errorMessage.
+     */
+    private updateAriaInvalid(): void {
+        const htmlInputField = this.comboboxDomRef.current?.getElementsByClassName('ms-ComboBox-Input').item(0);
+        if (htmlInputField) {
+            htmlInputField.ariaInvalid = this.props.errorMessage !== undefined ? 'true' : 'false';
```

Comment:
instead of manipulating directly with DOM, can we use props like here? -> https://github.com/SAP/open-ux-tools/blob/9d0e41b1a450d10b512904e4acc98451193f75e1/packages/ui-components/src/components/UIComboBox/UIComboBox.tsx#L705-L723

Reply:
like?:
```
private getAutofillProps(): IAutofillProps {
        const { highlight, readOnly, disabled, e } = this.props;
        const autofill: IAutofillProps = {};
        // Handle search highligh
        if (highlight) {
            autofill.onKeyDownCapture = this.onKeyDown;
        }
        const tabIndex = 'tabIndex' in this.props ? this.props.tabIndex : undefined;
        // Handle readOnly property
        if (readOnly && !disabled) {
            autofill.readOnly = readOnly;
            autofill.tabIndex = tabIndex;
            // Adjust aria attributes for readonly
            autofill['aria-disabled'] = undefined;
            autofill['aria-readonly'] = true;
        } else if (disabled) {
            autofill.disabled = undefined;
            autofill.readOnly = true;
            autofill.tabIndex = tabIndex;
        }
        if (this.props.errorMessage !== undefined) {
           autofill['aria-invalid'] = true;
        }
        return autofill;
    }
```

Reply:
@815are perfect, thanks, exactly what I needed!